### PR TITLE
Adding a jenkins role to the image_provisioner

### PR DESCRIPTION
**Pull request by mbruzek**
Currently the RHEL_OCP_3.7_QCOW_BUILDER has over one hundred lines of bash scripts to run remote commands via ssh. That code is not in source control and is not reviewable for correctness.

This PR converts that code to Ansible and puts it in source control so we can review the changes and improve incrementally.

Please review and let me know if you have any comments/questions or corrections. 

Thanks!

### CODE REVIEW

**Comment by mffiedler** [comment]
@chaitanyaenr Please review.

**Comment by chaitanyaenr** [comment]
@mbruzek did you test the role when atomic is set to true.

**Comment by mbruzek** [comment]
@chaitanyaenr No I did not build any atomic images with this code. We can do that after the cluster gets deployed next week.

**Comment by mbruzek** [comment]
@chaitanyaenr No I did not build a atomic image with this because I have no way to test if it is good or bad.

**Comment by chaitanyaenr** [comment]
@mbruzek I think we can just check if the image builder succeeds without failure for both rhel and atomic.

**Comment by mffiedler** [comment]
@mbruzek @chaitanyaenr Do we still want to merge this?   If so, please add a review approval.  Thanks.

**Comment by chaitanyaenr** [comment]
@mffiedler yes, this is ready for merge once we get it working for atomic as well.

File: image_provisioner/playbooks/roles/jenkins/templates/variables.yaml.j2 [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
```
Comment (chaitanyaenr) [inline image_provisioner/playbooks/roles/jenkins/templates/variables.yaml.j2]: @mbruzek can you update the inventory with the latest changes made to the image provisioner. https://github.com/openshift/svt/blob/master/image_provisioner/inventory/hosts.AWS can be used as a reference.

**Comment by mffiedler** [comment]
@mbruzek Please see request for changes.   Or should I close this out?

**Comment by chaitanyaenr** [comment]
LGTM except that the inventory file needs to be updated with the new vars.

**Comment by mffiedler** [comment]
@mbruzek Let's update the inventory file and get this ancient PR closed out :-)

**Comment by mbruzek** [comment]
I didn't mean to close this PR. Trying to reopen.

**Pull request closed without merging**